spec/requests/organisations_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'rails_helper'
+
+RSpec.describe "/organisations", type: :request do
+  fixtures :all
+
+  let(:valid_attributes) do
+    { name: 'Foo'  }
+  end
+
+  let(:invalid_attributes) do
+    { name: nil }
+  end
+
+  let(:user) { users :admin }
```

Comment:
The code `users :admin` is the syntax for grabbing a fixture. Basically, "give me the User object labelled 'admin'"